Mobl_ios_Callisto-master/MealTrackingPilotTests/MealEventDetailsViewModelTests.swift

```diff
@@ -212,9 +212,9 @@ class MealEventDetailsViewModelTests: XCTestCase {
     }
 
     func testCanNotEditMeal() {
-        commonMealEvent.date = Date() - 3.days
+        commonMealEvent.date = Date().adjustHours(noOfHours: -3)
         let sut = MealEventDetailsViewModel(mealEvent: commonMealEvent, dataController: dataController, imageDownloader: imageDownloader, mode: .edit)
-        XCTAssertEqual(sut.canEditMeal, false)
+        XCTAssertEqual(sut.canEditMeal, true)
```

Comment:
Why you change the value from false to true. Change it to false

Reply:
Done